Sources/GRPCProtobufCodeGen/ProtobufCodeGenParser.swift

```diff
@@ -15,7 +15,6 @@
  */
 
 internal import Foundation
-internal import SwiftProtobuf
```

Comment:
Was this import not used for anything?

Reply:
It was imported to access `NamingUtils` which is no longer used.

Reply:
`NamingUtils` comes from the `SwiftProtobufPluginLibrary`

Reply:
Oh yes. That means SwiftProtobuf was already not in use.